openstack/baremetalintrospection/v1/introspection/results.go

```diff
@@ -193,17 +195,22 @@ type CPUType struct {
 	ModelName    string   `json:"model_name"`
 }
 
+type LldpTlvType struct {
+	Type  int
+	Value string
+}
+
 type InterfaceType struct {
-	BIOSDevName string                 `json:"biosdevname"`
-	ClientID    string                 `json:"client_id"`
-	HasCarrier  bool                   `json:"has_carrier"`
-	IPV4Address string                 `json:"ipv4_address"`
-	IPV6Address string                 `json:"ipv6_address"`
-	Lldp        map[string]interface{} `json:"lldp"`
-	MACAddress  string                 `json:"mac_address"`
-	Name        string                 `json:"name"`
-	Product     string                 `json:"product"`
-	Vendor      string                 `json:"vendor"`
+	BIOSDevName string        `json:"biosdevname"`
+	ClientID    string        `json:"client_id"`
+	HasCarrier  bool          `json:"has_carrier"`
+	IPV4Address string        `json:"ipv4_address"`
+	IPV6Address string        `json:"ipv6_address"`
+	Lldp        []LldpTlvType `json:"lldp"`
```

Comment:
nit: `LLDP`

I realize this was originally `Lldp`, but it was missed during a prior review. Acronyms should be capitalized.

Reply:
Oh, cool. I prefer that but was trying to match the existing code. Will switch it to LLDP.